.github/bin/set-credentials

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+git config --global user.email "github-actions[bot]@users.noreply.github.com"
+git config --global user.name "github-actions[bot]"
```

Comment:
Should we use the exercism bot instead?

Reply:
The exercism bot? Do we have one?

The above code is the one we are currently using in the v3 workflows.